### ISSUE TITLE
Add download option for CERRA data

### ISSUE THREAD

**Issue by bvreede** [eWaterCycle/era5cli#120]
The CDS API also gives access to CERRA, see e.g. https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-cerra-model-levels?tab=overview

Adding this to era5cli would greatly elaborate the reach and use of the tool and make it future-proof.

(This could potentially mean separating the core of the tool from the specific era5 application. To be explored.)

**Comment by BSchilperoort** [comment]
I agree that supporting more CDS datasets is interesting and would make era5cli more generic. However I feel this would make the interface itself more complex, as it means every more keywords and options need to be supported and provided by the user.

Adding an overview of how the code is structured could already make it easier for others to re-use era5cli, and make their own cmip5cli or cerra_cli.
